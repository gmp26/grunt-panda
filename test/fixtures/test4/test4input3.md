````
title: test4input3
````
#test4input3

##subsection
_emphasis_
__bold__
$y=mx$
