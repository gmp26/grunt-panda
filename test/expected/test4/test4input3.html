<section id="test4input3" class="level1">
<h1>test4input3</h1>
<section id="subsection" class="level2">
<h2>subsection</h2>
<p><em>emphasis</em> <strong>bold</strong> <span class="math">\(y=mx\)</span></p>
</section>
</section>
